### ISSUE TITLE
Bulk CSV import ignores column header

### ISSUE THREAD

**Issue by sammystevens1983** [questdb/questdb#1758]
### Describe the bug

I am trying to stream data in via bulk csv import, since my data comes in big chunks.  The issue is my data will come in with a timing column, and a few data columns a/b/c, and the data columns can come in random order.
So with two files as below:
table 1

| timing | a   | b   | c   |
| ------ | --- | --- | --- |
| time1  | 1   | 0   | 0   |
| time2  | 1   | 0   | 0   |
| time3  | 1   | 0   | 0   |

table 2

| timing | c   | b   | a   |
| ------ | --- | --- | --- |
| time4  | 1   | 0   | 0   |
| time5  | 1   | 0   | 0   |
| time6  | 1   | 0   | 0   |

i see the following in questdb

| timing | a   | b   | c   |
| ------ | --- | --- | --- |
| time1  | 1   | 0   | 0   |
| time2  | 1   | 0   | 0   |
| time3  | 1   | 0   | 0   |
| time4  | 1   | 0   | 0   |
| time5  | 1   | 0   | 0   |
| time6  | 1   | 0   | 0   |

it appears to be ignoring column header, even playing with params doesnt seem to correct this issue.  It correctly sorts the time, so that if i send the latter portion, it puts it after the earlier data in questdb.  Just the columns are incorrect.

I am using the requests.post.  Applying and using params timestamp, overwrite, forceHeader, partitionby.

requests.post response says it correctly imported and no errors reported.  The odd thing is, it knows its sending C | B | A in that order, as it shows it in the response out of order.

it is on an air gapped system, posting code by hand.

### Expected Behavior

| timing | a   | b   | c   |
| ------ | --- | --- | --- |
| time1  | 1   | 0   | 0   |
| time2  | 1   | 0   | 0   |
| time3  | 1   | 0   | 0   |
| time4  | 0   | 0   | 1   |
| time5  | 0   | 0   | 1   |
| time6  | 0   | 0   | 1   |

### Environment

```markdown
- **QuestDB version**: 6.1.2
- **OS**: RHEL 7.9
- **Browser**: 
- **requests**: 2.11.1 
- **python**: 3.8.8
```

### Additional context

```python
import requests
data1 = open('data1.csv', 'rb').read()
data2 = open('data2.csv', 'rb').read()

payload = {'data': ('test', data1)}

response = requests.post(url='localhost:9000/imp', files=payload,
                         params={
    'timestamp': 'timing',
    'forceHeader': 'true',
    'partitionBy': 'MONTH',
    })

print response.text

payload = {'data': ('test', data2)}

response = requests.post(url='localhost:9000/imp', files=payload,
                         params={
    'timestamp': 'timing',
    'forceHeader': 'true',
    'partitionBy': 'MONTH',
    })

print response.text
```

may not work, i had to hand type since i couldn't copy/paste

**Comment by pswu11** [comment]
Hi @sammystevens1983 thanks a lot for raising the issue! I'd love to also invite you to our developer community on Slack to join different discussions on features/feedback: https://slack.questdb.io/

**Comment by ideoma** [comment]
Please put parameters in the URL in the `post` method

```
url='localhost:9000/imp?forceHeader=true
```

**Comment by sammystevens1983** [comment]
Same result, it just appends the unsorted columns.

**Comment by pswu11** [comment]
Hi @sammystevens1983 our team addressed this issue in our latest release 6.2. Let us know if it works for you now.
